docs/content/components/form/checkbox/checkbox.mdx

```diff
@@ -53,12 +49,12 @@ If you have number of options is moderate to large, typically more than 10-15 op
         width={700}
         height={700}
         unoptimized
-        src="/checkbox/checkbox-dont-select.jpg"
-        alt="Don't use checkboxes when only one item can be selected."
+        src="/checkbox/checkbox-dont-long-options.jpg"
+        alt="Don't use checkbox with long list of options."
       />
     </Dont.Figure>
     <Dont.Description>
-      Don't use checkboxes when only one item can be selected.
+      Don't use checkbox with long list of options.
```

Comment:
how long is long list of options?

Reply:
I just switched places. Same text as before :P

Reply:
but you also revising it now or not? Why not correct it?

Reply:
jup updated it